english/music-festival/lollapalooza-1997/lollapalooza-1997.md

```diff
@@ -0,0 +1,18 @@
+You leave work at Bell's Amusement Park in Tulsa OK and
+immediately drive non-stop to St. Louis.  The custom cherry
+red Datson 510 has no problem with the drive. 
+You crash on your sister's pull-out couch but can't sleep.
+You're too excited.
+
+Your day is a hazy blur of sight and sound.  The people are as
+deep as the sea.  In your fog you
+witness performances by:
+
+Orbital,
+Devo,
+The Prodigy,
+The Orb,
+Snoop Dog with Tupac,
+and the closing act, Tool
+
+Afterwards, you find yourself [holding the door](../hold_the_door/hodor.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.